examples/sample-app/README.md

```diff
@@ -462,10 +462,11 @@ Another interesting example is deleting a pod.
 
 Cleaning Up
 -----------
-To clean up all of your environment, you can run the script:
+To empty your project, deleting all BuildConfigs, ImageStreams, DeploymentConfigs,
+Services, etc, you can use:
 
-        $ sudo ./cleanup.sh
+    $ oc delete all --all
```

Comment:
this is not the equivalent to what cleanup was doing.

1) you're not stopping the openshift process
2) you need to run this as cluster-admin if you want it to really work and these instructions don't make that clear.